README.md

```diff
@@ -18,37 +18,37 @@ UIPs can be divided into the following categories:
 | Number                     | Title                     | Owner                          | Status    | Type      |
```

Comment:
Have to wonder if we shouldn't just have a `UIPS/index.md` instead of indefinitely growing the readme. (Which, by the way, should be `UIPs/`. (^: )

Reply:
I mean probably but Josh already did it this way and do we really want to change the README?

Reply:
You are changing the readme right now, brother.

But sure, if nothing else that'd be a separate PR.